src/main/ruby/core/truffle/interop.rb

```diff
@@ -27,13 +27,13 @@ def self.export_method(name)
 
     def self.object_keys(object)
       if object.is_a?(Hash)
-        object.keys.map do |key|
-          Truffle::Interop.to_java_string(key)
-        end
+        object.keys.map(&:to_s)
       else
-        object.instance_variables.map do |key|
-          Truffle::Interop.to_java_string(key[1..-1])
-        end
+        object.instance_variables.map { |i|
+          i = i.to_s
+          i = i[1..-1] if i.start_with?('@')
+          i
```

Comment:
I would name the block variable `ivar` here, `i` is usually am index or numeric.

Reply:
Fine.